fritz.coffee

```diff
@@ -143,18 +143,23 @@ module.exports = (env) ->
 
     # Initialize device by reading entity definition from middleware
     constructor: (@config, lastState, @plugin) ->
-      @id = config.id
-      @name = config.name
-      @interval = 1000 * (config.interval or plugin.config.interval)
+      @id = @config.id
```

Comment:
Yes. See http://coffeescript.org/#1.9.0

"Changed strategy for the generation of internal compiler variable names. Note that this means that @example function parameters are no longer available as naked example variables within the function body."